ArcGISRuntimeSDKQt_CppSamples/Layers/ExportTiles/ExportTiles.cpp

```diff
@@ -98,7 +98,7 @@ void ExportTiles::exportTileCacheFromCorners(double xCorner1, double yCorner1, d
 
     //! [ExportTiles start job]
     // execute the task and obtain the job
-    ExportTileCacheJob* exportJob = m_exportTileCacheTask->exportTileCache(m_parameters, m_tempPath.path() + "/offlinemap.tpk");
+    ExportTileCacheJob* exportJob = m_exportTileCacheTask->exportTileCache(m_parameters, m_tempPath.path() + "/offlinemap.tpkx");
```

Comment:
Does this for sure work? From the doc, it seems like it should, but not positive if you need to check if it supports compact v2 cache or not https://developers.arcgis.com/qt/cpp/api-reference/esri-arcgisruntime-exporttilecachetask.html

Reply:
It does run as expected after building both the individual sample and building the sample viewer?